src/msg/async/frames_v2.h

```diff
@@ -804,9 +804,12 @@ struct MessageFrame : public Frame<MessageFrame,
     return f;
   }
 
-  inline const ceph_msg_header2 &header() {
+  inline const ceph_msg_header2 header() {
```

Comment:
Since it is no longer a reference, `const` is useless and `inline` has always been redundant:
```suggestion
  ceph_msg_header2 header() {
```